app/policies/photo_policy.rb

```diff
@@ -0,0 +1,31 @@
+class PhotoPolicy < ApplicationPolicy
+  attr_reader :user, :photo
+
+  def initialize(user, photo)
+    @user = user
+    @photo = photo
+  end
+
+  def show?
+    user == photo.owner ||
+      !photo.owner.private? ||
+      photo.owner.followers.include?(user)
+  end
+
+  def edit?
+    user == photo.owner
+  end
+
+  def create?
+    user
+  end
+
+  def update?
+    user == photo.owner
```

Comment:
stay DRY

Reply:
called update? on the other methods to stay DRY